src/main/java/it/gov/pagopa/bizeventsservice/exception/ErrorHandler.java

```diff
@@ -142,6 +143,16 @@ public ResponseEntity<ProblemJson> handleAppException(final AppException ex, fin
         return new ResponseEntity<>(errorResponse, ex.getHttpStatus());
     }
 
+    @ExceptionHandler({FeignException.class})
+    public ResponseEntity<ProblemJson> handleFeignException(final FeignException ex, final WebRequest request) {
+        log.error("FeignException raised:", ex);
+        var errorResponse = ProblemJson.builder()
+                .status(HttpStatus.BAD_GATEWAY.value())
```

Comment:
I added this handler to return a 502 instead of 500 if there is a dependency fault.